jax/_src/xla_bridge.py

```diff
@@ -271,12 +252,13 @@ def make_cpu_client(
       collectives = xla_client._xla.make_mpi_collectives()
       collectives.Init()
       atexit.register(collectives.Finalize)
-    elif collectives_impl != 'none':
-      raise RuntimeError(f"Unknown collectives implementation "
-                        f"{collectives_impl}. Available implementations are "
-                        f"{CPU_COLLECTIVES_IMPLEMENTATIONS}.")
+    elif collectives_impl == 'megascale':
```

Comment:
You can't say megascale in OSS. There's separate handling for it internally.

Reply:
You can, we just don't expose code handling it. E.g. https://github.com/search?q=repo%3Ajax-ml%2Fjax%20megascale&type=code. And it's not sanitized from libtpu.